# FaintedGhost
It has come to me that a huge bug in the module "AutoArmour" is causing a NullPointerException when you leave a world - this patch is being worked on - thanks for your cooperation
